api/inventorypb/agents.proto

```diff
@@ -607,6 +607,21 @@ message GetAgentResponse {
   }
 }
 
+// Get logs
+
+message GetAgentLogsRequest {
```

Comment:
I think we need to to provide limit in LogsRequest, otherwise we can easily overload server and/or client

Reply:
We already have flag in CLI "log-lines-count" in https://github.com/percona/pmm/pull/926/files

Reply:
@ritbl `limit` done